unit-tests/live/d400/test-emitter-frequency.py

```diff
@@ -0,0 +1,63 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:device:jetson D457
+#test:device:!jetson D455
+
+import pyrealsense2 as rs
+from rspy import test, log, repo
+
+ctx = rs.context()
+device = test.find_first_device_or_exit();
+depth_sensor = device.first_depth_sensor()
+
+fw_version = repo.pretty_fw_version( device.get_info( rs.camera_info.firmware_version ))
+if repo.compare_fw_versions( fw_version, "5.13.1.53" ) <= 0:
+    log.i(f"FW version {fw_version} does not support EMITTER_FREQUENCY option, skipping test...")
+    test.print_results_and_exit()
+
+
+EMITTER_FREQUENCY_57_KHZ = 0.0
+EMITTER_FREQUENCY_91_KHZ = 1.0
+################################################################################################
+
+test.start("Verify camera defaults")
+device_nane = device.get_info(rs.camera_info.name)
+if "D455" in device_nane:
+    test.check_equal(depth_sensor.get_option(rs.option.emitter_frequency), EMITTER_FREQUENCY_57_KHZ)
+elif "D457" in device_nane:
+    test.check_equal(depth_sensor.get_option(rs.option.emitter_frequency), EMITTER_FREQUENCY_91_KHZ)
+else:
+    test.fail("Unexpected device name found: " + device_nane)
+test.finish()
+
+################################################################################################
+
+test.start("Test Set On/Off during idle mode")
+depth_sensor.set_option(rs.option.emitter_frequency,EMITTER_FREQUENCY_57_KHZ)
+test.check_equal(depth_sensor.get_option(rs.option.emitter_frequency), EMITTER_FREQUENCY_57_KHZ)
```

Comment:
does it pass without any sleep?

Reply:
Yes, XU commands are synchronic. this value is changing immediately

Reply:
ok